docs/checks.md

```diff
@@ -311,7 +311,8 @@ This check tries to determine if the project uses a dependency update tool,
 specifically one of:
 - [Dependabot](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates)
 - [Renovate bot](https://docs.renovatebot.com/configuration-options/)
-- [PyUp](https://docs.pyup.io/docs) (Python)
+- [PyUp](https://docs.pyup.io/docs) (Python) 
+
```

Comment:
this file is generated, take a look at the contributing MD for details of which file to edit:

https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#updating-docs